src/ZPublisher/tests/testHTTPRequest.py

```diff
@@ -749,7 +749,9 @@ def test_close_removes_stdin_references(self):
         s = BytesIO(TEST_FILE_DATA)
         start_count = sys.getrefcount(s)
 
-        req = self._makeOne(stdin=s, environ=TEST_ENVIRON.copy())
+        environ = TEST_ENVIRON.copy()
+        environ['CONTENT_LENGTH'] = str(len(TEST_FILE_DATA))
```

Comment:
We could, but what would the value be? The tests all use a different body, so the length value needs to be different.